packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -6,16 +6,13 @@
 // Usage:
 // `$ yarn create redwood-app ./path/to/new-project`
 
-import fs from 'fs'
 import path from 'path'
 
-import axios from 'axios'
 import chalk from 'chalk'
 import checkNodeVersion from 'check-node-version'
-import decompress from 'decompress'
 import execa from 'execa'
+import fs from 'fs-extra'
```

Comment:
I added this library which adds some missing methods to the `fs` library from the standard library. One of those methods is the method for copying directories.